qa/zcash/full_test_suite.py

```diff
@@ -126,13 +126,31 @@ def util_test():
         env={'PYTHONPATH': repofile('src/test'), 'srcdir': repofile('src')}
     ) == 0
 
+def rust_test():
+    target_dir = os.path.join(REPOROOT, 'target', 'x86_64-unknown-linux-gnu')
+    if not os.path.isdir(target_dir):
+        target_dir = os.path.join(REPOROOT, 'target', 'x86_64-apple-darwin')
```

Comment:
This will break once we start running tests on Windows and ARM. Open an issue to address this.